3p/environment.js

```diff
@@ -193,7 +193,8 @@ function instrumentEntryPoints(win) {
       next();
       if (typeof fn == 'string') {
         // Handle rare and dangerous string arg case.
-        return (0, win.eval/*NOT OK but whatcha gonna do.*/).call(win, fn);
+        const evil = win.eval;
```

Comment:
further testing shows doing `.call` and calling eval through `window` also triggers indirect invocation